### PR TITLE
Navbar

### LINKED ISSUE #25: Create navbar
Create navbar according to the wireframe: https://github.com/Abbe-and-Gabbe/jkpgCity/blob/main/JKPGCITY.pdf

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

File: frontend/src/Navbar.svelte [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+            display: inline-flex;
+            align-items: center;
+            cursor: pointer;
+        }
+
+        #toggler:checked ~ .menu {
+            max-height: 100%;
+        }
+    }
+</style>
```
Comment (albin02forsberg) [inline frontend/src/Navbar.svelte:71]: Move to  different file when css becomes this much. `Hamburger.svelte` is a good example when there is little css, then we can have it in the same file.

Comment (GabrielAdward) [reply frontend/src/Navbar.svelte:71]: Agreed, but waiting for tailwind. This is just temporary

Comment (albin02forsberg) [reply frontend/src/Navbar.svelte:71]: Working on this in issue #31 :D

Comment (albin02forsberg) [reply frontend/src/Navbar.svelte:71]: Added in #34, merge/rebase  with main when #34 in merged.

**Pull request closed without merging**